docker-compose.yml

```diff
@@ -1,15 +1,19 @@
 mongodb:
-    image: mongo:3.1
+    image: mongo:3.0
```

Comment:
Any reason for preferring an older mongo?

Reply:
@matthewma7 ?

Reply:
I don't know, it seems you changed it to 3.0 from 3.1. @aashish24